app/templates/components/flexberry-lookup.hbs

```diff
@@ -18,12 +18,12 @@
       readonly=(if (or readonly (not autocomplete)) "readonly")
     }}
     <button
-      class="ui {{if (or readonly isBlocked) " disabled"}} {{chooseButtonClass}} {{if (or modalIsBeforeToShow modalIsStartToShow) " loading"}} button"
+      class="ui {{if (or readonly isBlocked) " disabled"}} {{chooseButtonClass}} {{if (or modalIsBeforeToShow modalIsStartToShow) " loading"}} choose-button"
       {{action "choose" chooseData}}>
       {{{chooseText}}}
     </button>
     <button
-      class="ui {{if (or readonly isBlocked) " disabled"}} {{removeButtonClass}} button"
+      class="ui {{if (or readonly isBlocked) " disabled"}} {{removeButtonClass}} remove-button"
```

Comment:
CSS-classs 'button' must be restored, 'remove-button' class must be additional, but not replacing.